lib/autoupdate/agent/config.go

```diff
@@ -83,6 +87,16 @@ type UpdateStatus struct {
 	Skip *Revision `yaml:"skip,omitempty"`
 }
 
+// LastUpdate describes the last attempted updated.
+type LastUpdate struct {
+	// Success or failure of the attempted update
+	Success bool `yaml:"success"`
+	// Time the update occurred
+	Time time.Time `yaml:"time"`
+	// Target revision for the update
+	Target Revision `yaml:"target"`
+}
```

Comment:
Not sure how useful it would be to include the duration (update + health check) and error message when success is false, as this information can already be retrieved from the journal

Reply:
For now, this is just to support agent tracking on the server-side. We can add additional metadata later as its requested. The error message is a bit complicated, because multiple errors can occur during a revert, and the primary error may come from the sub-process.